force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -92,18 +112,28 @@ public with sharing class DummyJSONCallout {
      * @param contactId The Salesforce Contact ID used to generate the JSON payload for the external system.
      */
 	
-	public static void postCreateDummyJSONUser(String contactId) {
+	 @future(callout=true)
+	 public static void postCreateDummyJSONUser(String contactId) {
 		// Create HTTP request to send.
+		HttpRequest request = new HttpRequest();
 
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+		request.setEndpoint('callout:DummyJsonUser' + '/add');
 		// Set the HTTP method to POST.
-
+		request.setMethod('POST');
 		// Set the body using generateDummyJsonUserPayload method.
+		request.setBody(generateDummyJsonUserPayload(contactId));
 
 		// Send the HTTP request and get the response.
+		Http http = new Http();
+		HTTPResponse response = http.send(request);
 
 		// If the HTTP response code is successful, update the contact.
+		if (response.getStatusCode() >= 200 || response.getStatusCode() <= 299 ){
```

Comment:
```suggestion
		if (response.getStatusCode() >= 200 && response.getStatusCode() <= 299 ){
```
This should be AND here not OR. This only works here because of the = in the conditional